web/src/left_sidebar_navigation_area.js

```diff
@@ -85,7 +87,7 @@ function deselect_top_left_corner_items() {
 }
```

Comment:
How do stream items get deselected? I would kinda prefer to keep the DM implementation parallel to that, rather than paralleling the top-left items.

Reply:
would you prefer for me to add the `deselect_stream_items` there ?